game.rb

```diff
@@ -0,0 +1,19 @@
+require 'date'
+# create game class
+class Game
```

Comment:
class Game is inheriting from Item class if you see the diagram. So it should become `class Game< Item` . This means you dont have to rewrite the properties of Item class inside class Game, just use super(publish_date, archived)

Reply:
Thanks a lot, I'll do that